react/features/base/sounds/actions.js

```diff
@@ -74,6 +75,23 @@ export function playSound(soundId: string): Object {
     };
 }
 
+/**
+ * Stops playback of the sound identified by the given sound id.
+ *
+ * @param {string} soundId - The id of the sound to be stopped (the same one
+ * which was used in {@link registerSound} to register the sound).
+ * @returns {{
+ *     type: STOP_SOUND,
+ *     soundId: string
+ * }}
+ */
+export function stopSound(soundId: string): Object {
```

Comment:
To be honest I don't care, but it's a common expectation: this is not in alphabetical order :)